gremlin-go/README.md

```diff
@@ -22,58 +22,24 @@ for early testing purposes only.
 
 -->
 
-# Getting Started
```

Comment:
If we follow the pattern of the other README, we find they are fairly minimal and end user focused. Now that reference docs have been added for gremlin-go, I think it would be best to minimize the README contents to use end-user stuff, expand the reference/developer docs where necessary and then link to them from here.

Reply:
README is also used as package documentation https://pkg.go.dev/github.com/lyndonbauto/tinkerpop/gremlin-go/v3@v3.5.3-rc15